src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
@mbaudis comments?  I think metadata-working is closer, but can
you tell us for sure.

Maciek Smuga-Otto <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> > @@ -9,6 +9,32 @@ protocol Metadata {
> >  import idl "common.avdl";
> >
> >  /**
> > +  An ontology term describing an attribute. (e.g. the phenotype attribute
> > +  'polydactyly' from HPO)
> > +  */
> > +  record OntologyTerm {
> 
> There's both a metadata and a metadata-working branch, each of which has its
> own version of OntologyTerm. Which one should I settle on?
> 
> —
> Reply to this email directly or view it on GitHub.*
>